packages/viz/src/lib/style/helpers/color-bins-style.ts

```diff
@@ -0,0 +1,35 @@
+import { getColors, validateParameters } from './utils';
+
+export function colorBinsStyle(
+  featureName: string,
+  { bins = defaultOptions.bins, binColors = defaultOptions.binColors }: ColorBinStyleOptions = defaultOptions
+) {
+  validateParameters(featureName, bins, binColors);
+
+  const ranges = [...bins, Number.MAX_SAFE_INTEGER];
+  const rgbaColors = getColors(binColors, bins.length);
+
+  const getFillColor = (feature: Record<string, any>) => {
+    const featureValue: number = feature.properties[featureName];
+
+    // If we want to add various comparisons (<, >, <=, <=) like in TurboCARTO
+    // we can change comparison within the arrow function to a comparison fn
+    const rangeComparison = (definedValue: number, currentIndex: number, valuesArray: number[]) =>
+      (featureValue >= definedValue) && (featureValue < valuesArray[currentIndex + 1]);
```

Comment:
>= and <, I like it. We should probably document this behaviour in the function, as it is probably one of the most common confusions when dealing with bins

Reply:
Right now, it's static, it doesn't change. But what I wanted to note it's how it should change when adding that feature.

Reply:
Yes, perfect. I was just saying adding a small description about closed intervals to the function header

Reply:
Small detail: this filename uses color-bins-style.ts naming, should we go colorBinsStyle.ts ?

Reply:
I mixed naming conventions across files 🤦‍♂ 

Many people follow `-` separation and all lowercase, and it's the one that I like the most. But I need to change `defaultStyles.ts` to `default-styles.ts`.

What do you think?

Reply:
default-styles is better for reading, isn't it?